engines/wintermute/ad/ad_scene.cpp

```diff
@@ -1355,6 +1502,17 @@ bool AdScene::scCallMethod(ScScript *script, ScStack *stack, ScStack *thisStack,
 		return STATUS_OK;
 	}
 
+#ifdef ENABLE_WME3D
```

Comment:
One, possibly cleaner way to handle this mixture, is to introduce a separate function that implements the 3D-only functions, and returns true if it handled the function, false otherwise. That way you get a single #ifdef for all the functions.

Just a suggestion.

Reply:
let's postpone this for integration step with scummvm.

Reply:
Ok